priv/repo/migrations/20240612081138_add_auditing_settings.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Trento.Repo.Migrations.AddAuditingSettings do
+  use Ecto.Migration
+
+  def change do
+    alter table(:settings) do
+      add :auditing_retention_period, :integer
+      add :auditing_retention_period_unit, :string
```

Comment:
Something to consider: [default values](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#add/3-options) at the migration level.